examples/SPDXXMLExample-v2.0.xml

```diff
@@ -140,53 +149,282 @@ THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR IMPLIED WAR
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-      <relationshipType>COPY_OF</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>DocumentRef-spdx-tool-1.2:SPDXRef-ToolsElement</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-DOCUMENT</spdxElementId>
-      <relationshipType>DESCRIBES</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId> 
-</Relationship><Relationship>
-	  <spdxElementId>SPDXRef-CommonsLangSrc</spdxElementId>
-      <relationshipType>DYNAMIC_LINK</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-JenaLib</spdxElementId>
-      <relationshipType>DYNAMIC_LINK</relationshipType>
-      <comment></comment>
-      <relatedSpdxElementId>SPDXRef-Package</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-Package</spdxElementId>
-	  <relationshipType>DYNAMIC_LINK</relationshipType>
-	  <comment></comment>
-	  <relatedSpdxElementId>SPDXRef-Saxon</relatedSpdxElementId>
-</Relationship>
-<Relationship>
-	  <spdxElementId>SPDXRef-Package</spdxElementId>
-	  <relationshipType>CONTAINS</relationshipType>
-	  <comment></comment>
-	  <relatedSpdxElementId>SPDXRef-JenaLib</relatedSpdxElementId>
-</Relationship>
-</relationships>
-</Document></SpdxDocument>
\ No newline at end of file
+*/</extractedText>
+    <licenseId>LicenseRef-1</licenseId>
+  </hasExtractedLicensingInfo>
+  <annotation>
+    <annotationDate>2011-03-13T00:00:00Z</annotationDate>
+    <annotationType>REVIEW</annotationType>
+    <annotator>Person: Suzanne Reviewer</annotator>
+    <comment>Another example reviewer.</comment>
+  </annotation>
+  <annotation>
+    <annotationDate>2010-02-10T00:00:00Z</annotationDate>
+    <annotationType>REVIEW</annotationType>
+    <annotator>Person: Joe Reviewer</annotator>
+    <comment>This is just an example.  Some of the non-standard licenses look like they are actually BSD 3 clause licenses</comment>
+  </annotation>
+  <annotation>
+    <annotationDate>2010-01-29T18:30:22Z</annotationDate>
+    <annotationType>OTHER</annotationType>
+    <annotator>Person: Jane Doe ()</annotator>
+    <comment>Document level annotation</comment>
+  </annotation>
+  <documentDescribes>SPDXRef-Package</documentDescribes>
+  <documentDescribes>SPDXRef-File</documentDescribes>
+  <packages>
+    <comment>This package was converted from a DOAP Project by the same name</comment>
+    <copyrightText>NOASSERTION</copyrightText>
+    <downloadLocation>NOASSERTION</downloadLocation>
+    <filesAnalyzed>false</filesAnalyzed>
+    <homepage>http://www.openjena.org/</homepage>
+    <licenseConcluded>NOASSERTION</licenseConcluded>
+    <licenseDeclared>NOASSERTION</licenseDeclared>
+    <name>Jena</name>
```

Comment:
Same as JSON, it looks to me like the Packages / Files / Snippets are missing their SPDX ID definitions in the XML document also.

Reply:
Added SPDX ID's in the latest commit